_maps/map_files/IceBoxStation/IceBoxStation_skyrat.dmm

```diff
@@ -54483,6 +54483,16 @@
 /obj/effect/spawner/structure/window,
 /turf/open/floor/plating,
 /area/service/hydroponics/garden)
+"vkc" = (
+/obj/item/storage/belt/military/rocket/full,
+/obj/item/storage/belt/military/rocket/full,
+/obj/item/storage/belt/military/rocket/full,
+/obj/item/gun/ballistic/rocketlauncher/secuirty,
+/obj/item/gun/ballistic/rocketlauncher/secuirty,
```

Comment:
Bruh. Security misspelled

Reply:
*Secuirty, get it right Yawet

Reply:
That is what I get for no double-checking at 3:00am, lamo.